airbyte-integrations/connectors/source-facebook-marketing/Dockerfile

```diff
@@ -13,5 +13,5 @@ ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 
-LABEL io.airbyte.version=0.2.78
```

Comment:
You also need to update changelog in the doc file please

Reply:
updated